.changeset/mean-ears-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/generator": patch
+---
+
+Update Prisma (`prisma` and `@prisma/client`) from `4.6.0` to `4.6.1` to solve enum issue with postgresql https://github.com/prisma/prisma/issues/16180
```

Comment:
```suggestion
For new applications, update Prisma (`prisma` and `@prisma/client`) from `4.6.0` to `4.6.1` to solve enum issue with postgresql https://github.com/prisma/prisma/issues/16180
```

Reply:
To be honest, I don't understand what you what to say with "few" because AFAIK its all places that use the prisma package in this repo AFAIK. But I applied your changes, of course.

Reply:
@tordans because the only user facing change in this PR is generated apps. All the other changes are only internal to this repo, they don't affect end users. 

Does that makes sense?